lib/viewmodels/create_account_view_model.dart

```diff
@@ -31,18 +34,18 @@ class CreateAccountViewModel extends AsyncNotifier<CreateAccountValidation> {
       return "Cannot contain spaces.";
     }
 
-    if (value.length < 3) {
+    if (value.length < _minStringLength) {
       return "Too short.";
     }
 
-    if (value.length > 20) {
+    if (value.length > _maxStringLength) {
       return "Too long.";
     }
 
     // This regular expression is intentionally too restrictive (for instance,
     // the {3,20} is already checked above). Its purpose is too make sure that
```

Comment:
```suggestion
    // the {_minStringLength,_maxStringLength} is already checked above). Its purpose is too make sure that
```

Reply:
Good catch! Fixed in 083d1415676d38fe2fe3537cfd43208c7a280448